sockets/_base/socket_base.py

```diff
@@ -45,7 +45,12 @@ def draw(self, context, layout, node, text):
                     layout.label(text=text)
                 else:
                     layout.prop(self, "hide", icon='RADIOBUT_OFF', icon_only=True, invert_checkbox=True)
-                    layout.column().prop(node, self.default_prop, text=text)
+                    if self.default_prop == "in_selection_type":
```

Comment:
Better to use `self.default_type == "..."`
The prop name can be different, but the type would remain the same.

Reply:
Creating a new issue for this